openssl/Cargo.toml

```diff
@@ -11,7 +11,7 @@ path = "openssl.rs"
 path = "../crypto_bench"
 
 [dependencies]
-openssl = "0.6"
+openssl = "0.7"
```

Comment:
Thanks!

I think we should just change this to the git master like other benchmarks are already using. What do you think?